README.md

```diff
@@ -14,8 +14,59 @@ the packages individually, for example:
     make -C debs/xenial/archivematica
     make -C debs/xenial/archivematica-storage-service
 
-### Test packages
+#### Using parameters
 
+Most makefiles support parameters in the format of environment variables. They 
```

Comment:
I think "in the form of" sounds more natural than "in the format of" here.

Also, below your "BRANCH VERSION RELEASE" are just showing up on one line. I don't know if you maybe want them as an itemized list or in code highlighting.

Reply:
Done